pom.xml

```diff
@@ -133,7 +133,7 @@
             <groupId>org.jooq</groupId>
             <artifactId>jooq</artifactId>
             <!-- 3.14.0 broke returning() on MySQL :( -->
-            <version>3.13.5</version>
+            <version>3.15.12</version>
```

Comment:
Could you update the parent pom.xml instead and remove the version (and the comment) from here?